src/Model/PaymentMethodWithFeeTrait.php

```diff
@@ -20,7 +20,7 @@ trait PaymentMethodWithFeeTrait
     protected ?TaxCategoryInterface $taxCategory = null;
 
     /** @ORM\Column(name="calculator_configuration", type="json", nullable=true) */
-    protected array $calculatorConfiguration = [];
+    protected ?array $calculatorConfiguration = [];
```

Comment:
I think this will break bin/phpstan.sh ?

I see that circle-ci doesn't start when you do a PR from another repository.

Reply:
What do you mean it will break `bin/phpstan.sh`? :) 

Can you start the CI?